tests/system/Pipfile

```diff
@@ -1,22 +1,16 @@
 [[source]]
-
 name = "pypi"
 url = "https://pypi.python.org/simple"
 verify_ssl = true
 
-
 [requires]
-
 python_version = "3.6"
 
-
 [packages]
-
-shakedown = {git = "https://github.com/dcos/shakedown.git", ref = "1.5.0"}
+paramiko = "==2.4.2"
```

Comment:
All I did was `pipenv install paramiko==2.4.2` and the resulting diff in `Pipfile` and `Pipfile.lock` was a result of that.